src/groundStation/commandParser.py

```diff
@@ -152,12 +152,10 @@ def __argCheck(self, args, inoutInfo, subservice=None):
 
     def __lexer(self, input):
         tokenList = []
-<<<<<<< HEAD:src/commandParser.py
+        # '([a-zA-Z_-]+|[\(\)]|[0-9_.-]*)' #([^#]*)#|
         tmp = re.split('([-.]+|[0-9.]+|[a-zA-Z0-9_-]+|[\(\)])', input)
```

Comment:
Ok, we have had conversations about conflicts this regex create. I have tried a lot of different regexs (e.g., adding /S), and even though they're supposed to work, for some reason they don't parse and show errors. So, I'm keeping this one for now.

Reply:
A set of test cases I need to be considered as a single group: ab12x bc 4vc23 31.85 ---.--. -2 hello
And other regexes that are supposed to work but they don't:
([a-zA-Z_-]\S+|[\(\)]|[0-9_.-]\S*)
([-.]\S+|[0-9.]\S+|[a-zA-Z0-9_-]+|[\(\)])
([a-zA-Z0-9_.-]+|[\(\)]|[0-9_.-]*)